presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environment.java

```diff
@@ -116,13 +116,18 @@ public Builder addContainer(String name, DockerContainer container)
             checkState(!containers.containsKey(name), "Container with name %s is already registered", name);
             containers.put(name, requireNonNull(container, "container is null"));
 
-            String containerName = "ptl-" + name;
+            String containerName = "ptl-" + this.name + "-" + name;
             container
                     .withNetwork(network)
                     .withNetworkAliases(name)
                     .withLabel(PRODUCT_TEST_LAUNCHER_STARTED_LABEL_NAME, PRODUCT_TEST_LAUNCHER_STARTED_LABEL_VALUE)
                     .withCreateContainerCmdModifier(createContainerCmd -> createContainerCmd
                             .withName(containerName)
+                            // remove tc-xxxx random alias - it's not needed, and makes containers non-reusable
```

Comment:
where does this alias come from?

Reply:
from testcontainers when container is attached to a network (`withNetwork`)

testcontainers' `withReuse` doesn't work with networks: needs this fix here and `ReusableNetwork`